CONTRIBUTING.md

```diff
@@ -20,6 +20,9 @@ of this plugin:
     - Build the release package with `./ci/build_release`
     - Attach package to Github Release
 
+- To work on Development environment
+    - The repo includes dev and test scripts to setup environment in bin/dev 
```

Comment:
This should be moved out of the `## Releasing` section, and into its own section. It's important to include code samples for how a user can start the environment, and explain which services the scripts will start.

Check out the [Contributing doc in the cyberark/conjur repo](https://github.com/cyberark/conjur/blob/master/CONTRIBUTING.md#set-up-a-development-environment). Its `Set up a development environment` section is a pretty good example.